src/pages/index.js

```diff
@@ -66,16 +66,16 @@ const IndexPage = () => (
               children="1-week React bootcamp >>"
             />
           </Col>
-          <Col xs={12} sm={4}>
+          <Col xs={12} sm={3}>
             <LinkButton
-              to="/react-native-bootcamp"
-              children="React Native training >>"
+              to="/advanced-react-redux-graphql-bootcamp "
```

Comment:
@shakersinthedark this link is broken when I click on it because of the blank space at the end